server/routes/handlers/outcomes/getCasesInProgressHandler.js

```diff
@@ -7,19 +9,25 @@ const getCasesInProgressHandler = caseService => async (req, res) => {
     session
   } = req
 
-  const filters = getOutcomeTypesListFilters(req.query)
-  const filtersApplied = filters.map(filterObj => filterObj.items.filter(item => item.checked).length).pop()
+  const response = await caseService.getOutcomesList(courtCode, req.query, sorts, state)
+
+  const cases = response.cases
```

Comment:
Could use destructuring like line 6 but I'm not going to reject a PR for that

Reply:
done